test/blackbox-tests/test-cases/github25/run.t

```diff
@@ -0,0 +1,27 @@
+This test define an installed "plop" with a "plop.ca-marche-pas"
+sub-package which depend on a library that doesn't exist.
+
+The build itself uses only "plop.ca-marche", which doesn't have this
+problem. So jbuilder shouldn't crash because of "plop.ca-marche-pas"
+
+We need ocamlfind to run this test
+
+  $ jbuilder build @install --display short --only hello
+  File "root/jbuild", line 5, characters 14-28:
+  Error: Library "plop.ca-marche" not found.
+  Hint: try: jbuilder external-lib-deps --missing --only-packages hello @install
```

Comment:
Is this change expected?

Reply:
Yeah, it's to get rid of the `chdir` in the test runner. What was the purpose of the original `chdir`, seems like the test works the same way if we don't change the root.

Reply:
@diml is the above ok?

Reply:
Sorry, my review comment got lost...

Basically the error shouldn't be there. In the old test there was a `(setenv OCAMLPATH ../findlib ...)`, did you keep something similar? Alternatively, we can just set the variable here directly for simplicity

Reply:
I can't see the commit :(

Reply:
Oops. Check again.